### PR TITLE
Gsap devtools

### PR DESCRIPTION
**Pull request by jeremybarbet**
It's kinda workin-ish for the moment, which mean, it's super bad.

- Animations that are not autoplayed, are not track by the gsdevtools
- We can't only pass a wanted animation to the gsdevtools, everything that's autoplayed is tracked
- Avoid devtools to be created multiple times
- Update the animations list for animations that wasn't autoplayed before

### CODE REVIEW

File: shared/components/gsap-tools/Wrapper.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+
+  componentDidMount() {
+    // Update the GSDevTools to add the decorated component
+  }
+
+  render() {
+    const { children } = this.props;
+
+    return Children.map(children, (c) => {
```
Comment (birkir) [inline shared/components/gsap-tools/Wrapper.js:17]: React.Children.only ?

File: shared/components/gsap-tools/Wrapper.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      if (c) {
+        return cloneElement(c, {
+          ref: (el) => { this.children = el; },
```
Comment (birkir) [inline shared/components/gsap-tools/Wrapper.js:20]: Overwriting this.children in the loop

**Review by birkir — COMMENTED** [review]
Haven't tried it yet but looks promising! 🎉

**Comment by jeremybarbet** [comment]
Well, I was just experimenting some stuff around the official gsap devtools, but it doesn't really fit our needs, so we will just create our own tool, ouyouyou!

**Pull request closed without merging**